src/test/java/uk/co/ramp/covid/simulation/place/ConstructionSiteTest.java

```diff
@@ -22,11 +25,56 @@ public void setupParams() throws IOException {
     }
 
     @Test
-    public void testConstructionSiteTransProb() throws JsonParseException, IOException {
+    public void testConstructionSiteTransProb() throws JsonParseException {
         RNG.seed(123);
         ConstructionSite constructionSite = new ConstructionSite();
         double expProb = PopulationParameters.get().getpBaseTrans() * 10d / (5000d / 100d);
         double delta = 0.01;
         assertEquals("Unexpected construction site TransProb", expProb, constructionSite.transProb, delta);
     }
+
+    @Test
+    public void testNoConstructionSites() throws JsonParseException {
+        //The input ConstructionSites ratio is set very high so that there are no construction sites.
+        //Check that each person's primary place is never set to construction site
+        PopulationParameters.get().setConstructionSiteRatio(100000);
```

Comment:
We should possibly add cases to handles the ratio just being 0? I think that's clearer than trying to force it this way (and is possibly going to be used in the parameters depending which place we are modelling)

Reply:
Setting a ratio of 0 gives a divide by zero error.